src/Common/Tooltip/types.ts

```diff
@@ -21,9 +36,34 @@ type BaseTooltipProps =
           showOnTruncate?: never
           /**
            * If true, wrap with tippy irrespective of other options
-           * @default false
+           * @default true
            */
           alwaysShowTippyOnHover?: boolean
+          /**
+           * If true, use the common styling for shortcuts
+           * @default false
+           */
+          shortcutKeyCombo?: never
+      }
+    | {
+          /**
+           * If true, show tippy on truncate
+           * @default false
+           */
+          showOnTruncate?: never
+          /**
+           * If showOnTruncate is defined this prop doesn't work
+           * @default false
+           */
+          alwaysShowTippyOnHover?: never
```

Comment:
Please add a todo for this since we should be able to configure this, for e.g, disabled state

Reply:
Allowing user to pass alwaysShowTippyOnHover along with shortcutKeyCombo